src/Messages.js

```diff
@@ -42,11 +42,9 @@ export default defineMessages({
     description: 'Sig table column title',
     defaultMessage: 'Last matched',
   },
-  newMatchCount: {
-    id: 'newMatchCount',
-    description: 'Sig table column title for New matches',
-    defaultMessage: 'New matches',
-  },
+  newMatchCount: 'New matches',
+  newMatchesColumnTooltip:
+    'This indicates a new, unreviewed malware match detected within the last 30 days.',
```

Comment:
These two messages are defined and used incorrectly, instead of the object values being strings, they should be objects with keys `id`, `description` and `defaultMessages` and instead of consuming them directly with `messages.newMatchCount`, they should be intl injected like so `intl.formatMessage(messages.name)`. `intl` object can be fetched with `const intl = useIntl();`